view.py

```diff
@@ -1,20 +1,293 @@
 import tkinter as tk
 from tkinter import ttk
 
+TITLEFONT = ("Time New Roman", 15)
 class View(tk.Tk):
-	def __init__(self, controller):
-		'''
-		Constructor 
-		'''
-		super().__init__() # call init of tk.Tk parents
-		self.controller = controller 
-		
-
-	def main(self):
-		self.mainloop()
-
-	def _create_components(self):
-		'''
-		Put the elements here
-		'''
-		pass
+
+    # __init__ function for class tkinterApp
+    def __init__(self, controller):
+        
+        self.controller = controller
+
+        # __init__ function for class Tk
+        tk.Tk.__init__(self)
+        self.geometry("500x300")
+        self.state("zoomed")
+        self.columnconfigure(0, weight=1)
+        self.rowconfigure(0, weight=1)
+        
+        # creating a container
+        container = tk.Frame(self)
+        container.grid(column = 0, row = 0)
+
+        self.user_info = {}
+        self.user_defined_meal = []
+
+        # initializing frames to an empty array
+        self.frames = {}
+
+        # iterating through a tuple consisting
+        # of the different page layouts
+        for F in (CreateNutritionPlan, CreateBodyCondition, CreatePlanPeriod, CreateUserDefinedMeal):
+
+            frame = F(container, self)
+
+            # initializing frame of that object from
+            # CreateNutritionPlan, CreateBodyCondition, #CreatePlanPeriod respectively with
+            # for loop
+            self.frames[F] = frame
+
+            frame.grid(row = 0, column = 0, sticky ="nsew")
+        
+        self.user_info["Nutrition"] = self.frames[CreateNutritionPlan]
+        self.user_info["Body"] = self.frames[CreateBodyCondition]
+        self.user_info["Plan"] = self.frames[CreatePlanPeriod]
+
+        self.show_frame(CreateNutritionPlan)
+
+        
+
+    # to display the current frame passed as
+    # parameter
+    
+    def show_frame(self, current_frame):
+        frame = self.frames[current_frame]
+        frame.tkraise()
+
+class CreateNutritionPlan(tk.Frame):
```

Comment:
I wonder if the name can be only `NutritionPlan`. Generally, a class name should be a noun.

Reply:
^ The above comments are applied for similar classes below.

Reply:
Also, I think we should move the classes that is not `View` class into another file. This will help better organize them, and make this file more coherent.